### ISSUE TITLE
2.0 has a test for "Invalid Tweet: 141 characters"

### ISSUE THREAD

**Issue by dequis** [twitter/twitter-text#208]
https://github.com/twitter/twitter-text/blob/34dc1dd9f10e9171100cdff0cb2b7a9ed9ea2bd6/conformance/validate.yml#L28-L30

Is that an oversight? Or do tests pass with this?

**Comment by kaushlakers** [comment]
Hi @dequis. Tests still pass because we still support the old way of counting and they're configured to run that way. It's based on the config defined here https://github.com/twitter/twitter-text/tree/master/config

We will probably get rid of them soon. Thoughts @sudhee @dlamacchia @leeaustinadams ? On when we want to do this.

**Comment by dlamacchia** [comment]
Yeah, this could stand to be cleaned up. As long as we support v1, we might just move these tests to a v1 section and clarify why they're still there.
